doc/source/querying.rst

```diff
@@ -0,0 +1,96 @@
+
+In general, these data structures operate by constructing them using one set of points, after which they can be queried to efficiently find the neighbors of arbitrary other points using :py:meth:`freud.locality.NeighborQuery.query`.
+
+
+Query Arguments
+===============
+
+The table below describes the set of valid query arguments.
+
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| Query Argument | Definition                                                            | Data type | Legal Values              | Valid for                                                           |
++================+=======================================================================+===========+===========================+=====================================================================+
+| mode           | The type of query to perform (distance cutoff or number of neighbors) | str       | 'none', 'ball', 'nearest' | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| r_max          | Maximum distance to find neighbors                                    | float     | r_max > 0                 | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| r_min          | Minimum distance to find neighbors                                    | float     | 0 <= r_min < r_max        | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| num_neighbors  | Number of Neighbors                                                   | int       | num_neighbors >= 0        | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
++----------------+-----------------------------------------------------------------------+-----------+---------------------------+---------------------------------------------------------------------+
+| exclude_ii     | Whether or not to include self-neighbors                              | bool      | True/False                | :class:`freud.locality.LinkCell`, :class:`freud.locality.AABBQuery` |
```

Comment:
"self-neighbors" isn't clear to me - it's excluding the same ids, not necessarily the same positions.